sdk/package.json

```diff
@@ -25,8 +27,12 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "@types/chai": "^4.3.11",
```

Comment:
It looks like some conflict in the IDE
![image](https://github.com/invariant-labs/protocol-a0/assets/43304803/bec6ea1f-a6ef-4e25-94ff-7eac6825aefd)